src/server/ServerHandler.java

```diff
@@ -197,7 +195,39 @@ private GamingResponse handleRequestMove() {
      */
     @Override
     public void run() {
-        // Empty for now - will process client requests later
+        while (true) {
+            try {
+                // Read serialized request from client
+                String serializedRequest = dataInputStream.readUTF();
+                LOGGER.info("Received request: {}", serializedRequest);
```

Comment:
Change this to debug logging

```suggestion
                LOGGER.debug("Received request: {}", serializedRequest);
```